core/src/main/scala/tech/beshu/ror/accesscontrol/ops.scala

```diff
@@ -156,9 +157,16 @@ object show {
           showTraversable("response_hdr", bc.responseHeaders) ::
           showTraversable("repositories", bc.repositories) ::
           showTraversable("snapshots", bc.snapshots) ::
+          showTraversable("response_transformations", bc.responseTransformations) ::
```

Comment:
it will look like this:
```
... RESOLVED:[...,response_transformations=[FilteredResponseFields(fields=[f1,f2],mode=blacklist)]] ...
```

what about sth like that:
```
... RESOLVED:[...,response_trasformations=[fields=[~f1,~f2]]] ...
```

WDYT?

Reply:
Indeed it looks more concise, I would probably use different name for `fields` to make it more clear in what it does. I thought about `filtered_fields`. However I don't know if there aren't too many names for the same thing like "response fields", "filtered fields". WDYT?

Reply:
I've intentionally skipped the `response` part. I though like that:
* we have `response_transformations` array
* in that array we found one transformation which is [response] fields [transformation] 
so, IMO the context is full there.